ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,63 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test, handle_frontend_function
+
+
+# asarraya
+# TODO: dtype testing should be independent of input_dtype
+@handle_frontend_test(
+    fn_tree="numpy.asarray",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
+)
+def test_numpy_asarray(
+    *,
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=x,
+        dtype=input_dtype[0],
+    )
+
+
+@handle_frontend_function(
```

Comment:
This decorator `@handle_frontend_function` no longer exists. Only `@thandle_frontend_test` should be used here.

Reply:
This was what I did before. Now, I got a message that the task is stale. I don't really understand. I did everything according to your instructions.